### PR TITLE
Add support for partial orders, closes #46

### LINKED ISSUE #46: Support partial orders in frontend
Backend allows partial fills but frontend doesn't.

### PR DESCRIPTION
**Pull request by NTPape**

### CODE REVIEW

File: frontend/client/templates/offermodal.js [hunk 1]
```diff
@@ -1,20 +1,99 @@
...
+      this.volume(volume.toString(10))
+      this.total(total.toString(10))
+    }
+  },
+  maxVolume: function () {
     try {
-      return web3.fromWei(new BigNumber(price)).times(new BigNumber(volume)).toString(10)
+      if (Template.currentData().offer.type === 'bid') {
+        // Calculate max volume, since we want to sell MKR, we need to check how much MKR we can sell
+        var token = Tokens.findOne('MKR')
```
Comment (rainbeam) [inline frontend/client/templates/offermodal.js]: Not sure you meant to hardcode MKR here - shouldn't it be `Template.currentData().offer.currency`?

Comment (NTPape) [inline frontend/client/templates/offermodal.js]: Bid offers are about selling MKR, and the volume is in MKR, so for the maximal possible volume the MKR balance and allowance are relevant.

Comment (rainbeam) [inline frontend/client/templates/offermodal.js]: Ah, ok. Should it not be `BASE_CURRENCY` for consistency then?

Comment (NTPape) [inline frontend/client/templates/offermodal.js]: Yes, you're right.

**Pull request merged**